src/applications/disability-benefits/all-claims/tests/fixtures/data/maximal-toxic-exposure-test.json

```diff
@@ -0,0 +1,348 @@
+      "state": "NY",
+      "zipCode": "54321",
+      "effectiveDate": {
+        "from": "2099-12-01",
+        "to": "2100-01-01"
+      }
+    },
+    "view:contactInfoDescription": {},
+    "additionalDocuments": [
+      {
+        "name": "lolwut.png",
+        "confirmationCode": "552067b2-9c5d-4bd8-bcd8-bc621b51a145",
+        "attachmentId": "L015"
+      },
+      {
+        "name": "image (1).png",
+        "confirmationCode": "0496e5c2-0675-4a43-83d6-e1eeabd4ea0e",
+        "attachmentId": "L070"
+      },
+      {
+        "name": "Success.jpg",
+        "confirmationCode": "de89492d-2d3f-4486-a73d-1fa4868aa49d",
+        "attachmentId": "L023"
+      }
+    ],
+    "view:uploadPrivateRecordsQualifier": {
+      "view:aboutPrivateMedicalRecords": {},
+      "view:hasPrivateRecordsToUpload": true,
+      "view:privateRecordsChoiceHelp": {}
+    },
+    "privateMedicalRecordAttachments": [
+      {
+        "name": "AngleSettingJig_9_25_15.pdf",
+        "confirmationCode": "544f14cc-4e51-4661-b8f4-d79f71567491",
+        "attachmentId": "L107"
+      },
+      {
+        "name": "image.png",
+        "confirmationCode": "fa0c0f47-a42f-4bbc-88f3-873c5bea6ce7",
+        "attachmentId": "L023"
+      }
+    ],
+    "view:vaMedicalRecordsIntro": {},
+    "vaTreatmentFacilities": [
+      {
+        "treatmentCenterName": "Treatment Center the First",
+        "treatmentDateRange": {
+          "from": "2008-01-XX"
+        },
+        "treatmentCenterAddress": {
+          "country": "USA",
+          "city": "Bigcity",
+          "state": "AK"
+        },
+        "treatedDisabilityNames": {
+          "diabetesmellitus0": true,
+          "diabetesmellitus1": true,
+          "heartattackmyocardialinfarction": true
+        }
+      },
+      {
+        "treatmentCenterName": "Treatment Center the Second",
+        "treatmentDateRange": {
+          "from": "2010-01-XX"
+        },
+        "treatmentCenterAddress": {
+          "country": "Uganda",
+          "city": "Ugandan City"
+        },
+        "treatedDisabilityNames": {
+          "asthma": true,
+          "heartattackmyocardialinfarction": true,
+          "ankylosisinkneebilateral": true
+        }
+      }
+    ],
+    "view:hasEvidence": true,
+    "view:hasEvidenceFollowUp": {
+      "view:selectableEvidenceTypes": {
+        "view:hasVaMedicalRecords": true,
+        "view:hasPrivateMedicalRecords": true,
+        "view:hasOtherEvidence": true
+      },
+      "view:evidenceTypeHelp": {}
+    },
+    "view:unemployable": false,
+    "view:aidAndAttendance": true,
+    "view:modifyingHome": true,
+    "view:modifyingCar": true,
+    "view:needsCarHelp": {
+      "view:alreadyClaimedVehicleAllowance": false
+    },
+    "toxicExposureConditions": {
+      "asthma": true
+    },
+    "gulfWar1990": {
+      "airspace": true
+    },
+    "gulfWar1990Locations": {
+      "afghanistan": {
+        "startDate": "1990-01-01",
+        "endDate": "1990-12-31"
+      }
+    },
```

Comment:
This json fixture was copied from `src/applications/disability-benefits/all-claims/tests/fixtures/data/maximal-test.json` and then lines 139 - 150 added for toxic exposure

Reply:
I can add another location that has another set of start and/or end dates, but i do also want to make sure we cover the scenario where a Veteran selects a location, but doesn't recall the dates (which is what's currently being done for `"airspace"` - the dates are optional).

At some point when we switch from `va-memorable-date` to `va-date`, we should also check the scenario for year only

Reply:
fyi added `iraq` with dates